honeybee/face.py

```diff
@@ -548,14 +548,65 @@ def apertures_by_ratio_rectangle(self, ratio, aperture_height, sill_height,
             aperture._parent = self
             self._apertures.append(aperture)
 
+    def apertures_by_width_height_rectangle(self, aperture_height, aperture_width,
+                                            sill_height, horizontal_separation,
+                                            tolerance=0):
+        """Add repeating apertures to this face given the aperture width and height.
+
+        Note that this method removes any existing apertures on the Face.
+
+        Note that this method will effectively fill any rectangular portions of
+        this Face with apertures at the specified width, height and separation.
+        If no rectangluar portion of this Face can be identified, no apertures
+        will be added.
+
+        Args:
+            aperture_height: A number for the target height of the apertures.
+            aperture_width: A number for the target width of the apertures.
+            sill_height: A number for the target height above the bottom edge of
+                the rectangle to start the apertures. If the aperture_height
+                is too large for the sill_height to fit within the rectangle,
+                the aperture_height will take precedence.
+            horizontal_separation: A number for the target separation between
+                individual apertures centerlines.  If this number is larger than
+                the parent rectangle base, only one aperture will be produced.
+            tolerance: The maximum difference between point values for them to be
+                considered a part of a rectangle.
+
+        Usage:
+            room = Room.from_box(3.0, 6.0, 3.2, 180)
+            room[1].apertures_by_ratio_rectangle(0.4, 2, 0.9, 3)
```

Comment:
This is not related to this PR but just seeing this I feel we should use `room.faces[1]` in our sample code which is more explicit way to access the faces.

Reply:
Ok. I'm glad that you pointed this out since I also just realised that this usage example is not correct (it seems that I forgot to change it after copy/pasting it from another docstring). I'll fix it now.

Reply:
Done!